e2e/tests/ui_tests/lrspec/solicitorCreatesClaim-LRspec_test.js

```diff
@@ -51,79 +51,79 @@ const journeys = {
 
 Feature('Claim creation @e2e-tests-spec');
 
-Scenario('Applicant solicitor creates specified claim individual-to-individual @create-claim-spec', async ({I}) => {
+Scenario.skip('Applicant solicitor creates specified claim individual-to-individual @create-claim-spec', async ({I}) => {
```

Comment:
Why are we skipping these tests? I am planning to run lr tests in demo. If we skip it might not run anything?